app/templates/index.hbs

```diff
@@ -20,7 +20,7 @@
               class="lg:col-3 bg-dark"
               @href="{{repo.htmlUrl}}/issues?q=is%3Aissue+is%3Aopen+label%3A%22Help+Wanted%22+sort%3Acreate-date"
               @title={{repo.displayName}}>
-              <p>Forks: {{#if (not repo.forks)}} No forks {{else}} {{repo.forks}} {{/if}}</p>
+              <p>Forks: {{if repo.forks repo.forks "No forks"}}</p>
```

Comment:
The line definitely reads nicer. We can assume that GitHub always return a number for `repo.forks` (i.e. not `undefined`) so we could simplify to:

```suggestion
              <p>Forks: {{repo.forks}}</p>
```